### ISSUE TITLE
Document what to do after `composer drupal-init`

### ISSUE THREAD

**Issue by marcvangend** [hussainweb/drupal-composer-init#3]
Hi, thanks for drupal-composer-init, I really like it!

Maybe the documentation can be improved a little. I'm not a newbie when it comes to composer or Drupal, but still after running `composer drupal-init` I found myself thinking: "That was it? Where is my site? What do I do now?" Already suspecting that I had to run `composer install`, I checked the documentation for confirmation... but alas.

TL;DR: Please add in the "Usage" section of README.md that one should run `composer install` too.
